cmd/devices-api/api.go

```diff
@@ -239,6 +239,7 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, pdb db.Store,
 	udOwner.Delete("/", userDeviceController.DeleteUserDevice)
 	udOwner.Get("/commands/mint", userDeviceController.GetMintDevice)
 	udOwner.Post("/commands/mint", userDeviceController.PostMintDevice)
+	udOwner.Post("/commands/mint-with-sacd", userDeviceController.PostMintDeviceWithSACD)
```

Comment:
wombo combo these bad boys

Reply:
I, too, think it's okay to keep the same endpoint.

Not that you couldn't do this even with your setup, but I think it would also encourage re-using the code.

Reply:
yep, I agree-- the original ticket said to do it in two different endpoints, which is why I took this approach initially